<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kelime Öğrenme Kartı</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../usr/kde/3.5/share/icons/crystalsvg/32x32/apps/abiword.png</normaloff>../../../../../../usr/kde/3.5/share/icons/crystalsvg/32x32/apps/abiword.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>../..</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="lbKart">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButton">
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Pencereyi Kapat</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="ikonlar.qrc">
       <normaloff>:/ikonlar/ikonlar/agt_stop.png</normaloff>:/ikonlar/ikonlar/agt_stop.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btCevir">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Kartı Çevir</string>
     </property>
     <property name="icon">
      <iconset resource="ikonlar.qrc">
       <normaloff>:/ikonlar/ikonlar/kartmodu2.png</normaloff>:/ikonlar/ikonlar/kartmodu2.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="digerKart">
     <property name="text">
      <string>Sonraki</string>
     </property>
     <property name="icon">
      <iconset resource="ikonlar.qrc">
       <normaloff>:/ikonlar/ikonlar/1rightarrow.png</normaloff>:/ikonlar/ikonlar/1rightarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pBOnceki">
     <property name="text">
      <string>Önceki</string>
     </property>
     <property name="icon">
      <iconset resource="ikonlar.qrc">
       <normaloff>:/ikonlar/ikonlar/player_rew.png</normaloff>:/ikonlar/ikonlar/player_rew.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pBOnceki</tabstop>
  <tabstop>digerKart</tabstop>
  <tabstop>btCevir</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <resources>
  <include location="ikonlar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
